Remove --isolated from flake8 tool

sargunv opened the pull request:
Context: #221 

According to `flake8 --help`, looks like `--config` already ignores configs other than the one provided, so `--isolated` is redundant.
```
  --config=CONFIG       Path to the config file that will be the authoritative
                        config source. This will cause Flake8 to ignore all
                        other configuration files.
  --isolated            Ignore all found configuration files.
```

Additionally, the `--isolated` flag ignores *all* configs, not just found ones, so it renders the config file passed with `--config` useless.

https://github.com/PyCQA/flake8/blob/master/src/flake8/options/config.py#L295-L306

markstory commented on lintreview/tools/flake8.py:
```diff
@@ -60,7 +60,7 @@ def process_files(self, files):
         process_quickfix(self.problems, output, docker.strip_base)
 
     def make_command(self, files):
-        command = ['flake8', '--isolated']
```
There were a number of problems without `--isolated` mode as flake8 would pick up options from `setup.cfg`, and that file would sometimes modify the output format making it impossible to be reliably parsed.

This flag *could* be removed if the `config` flag is present, but its still kind of risky as any config file with a formatter in it will break review results.

markstory replied:
Don't we need `--format=default` to be in the command options now?

sargunv commented:
Sorry for the delay.

I've modified the PR so `--isolated` is included unless `config` is specified, then `--format default` is included. I also updated the relevant test and added one new test.

sargunv commented:
Thanks for merging! How long does it typically take for an update to master to make it into Stickler CI?

markstory commented:
It is a manual process. I hope to get to it in the next day or so.
